.github/workflows/ci-run-test.yml

```diff
@@ -214,6 +214,8 @@ jobs:
       - name: Install Python deps
         run: |
           python -m pip install --upgrade pip requests wheel urllib3 chardet
+      - name: Install poetry
+        run: pipx install poetry
```

Comment:
Where does `pipx` come from?  Can we assume the actions image contains that, or should we add it to the python dependencies above?

Reply:
Yeah, runner images always include a preinstalled pipx

https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md
https://github.com/actions/runner-images/blob/main/images/macos/macos-14-arm64-Readme.md